sassoftware/vscode-sas-extension
# feat: support formatting SAS code

**Summary**
Resolve #28

**Testing**

1. Open a `.sas` file
2. Open context menu and select `Format Document`
3. See formatted SAS code
